main/createUiDefinition.json

```diff
@@ -227,7 +193,6 @@
       }
     ],
     "outputs": {
-      "customData": "[basics('customData')]",
```

Comment:
@andrei-maroz   should we remove ssh as well in release branch ::         "constraints": {
              "allowedValues": [
                {
                  "label": "Bootstrap UI (443)",
                  "value": "bootstrapUi"
                },
                {
                  "label": "SSH (22)",
                  "value": "ssh"
                }

Reply:
I removed SSH rule for SG.